test/integration/install.test.js

```diff
@@ -162,11 +162,7 @@ describe('snowpack install', () => {
       // If any diffs are detected, we'll assert the difference so that we get nice output.
       for (const entry of res.diffSet) {
         // NOTE: We only compare files so that we give the test runner a more detailed diff.
-        if (entry.type1 !== 'file') {
-          return;
-        }
-        // NOTE: common chunks are hashed, non-trivial to compare
-        if (entry.path1.endsWith('common') && entry.path2.endsWith('common')) {
+        if (entry.type1 === 'directory' && entry.type2 === 'directory') {
```

Comment:
This is the major diff here. The other added files were obscured from this being missing.